static/styles/left_sidebar.css

```diff
@@ -62,6 +62,52 @@ li.show-more-topics {
     }
 }
 
+#show_more_pms {
+    cursor: pointer;
+    padding-top: 1px;
+    padding-bottom: 1px;
+
+    a {
+        font-size: 12px;
+        padding-left: 32px;
+    }
+
+    .unread_count {
+        margin-right: 30px;
+        margin-top: 2px;
+    }
+}
+
+#all_private_messages_icon {
+    opacity: 0.7;
+    display: inline;
+    right: 0;
+    position: absolute;
+    margin-right: 24px;
+    margin-top: 3px;
+
+    &:hover {
+        opacity: 1;
+        cursor: pointer;
+    }
+}
+
+#toggle_private_messages_section {
+    width: 90%;
+}
+
+#toggle_private_messages_section_icon {
```

Comment:
I think we want this CSS to be for a  `.toggle_sidebar_section_icon` class, since I imagine we're going to end up wanting to reuse this toggling feature for other sections.